### ISSUE TITLE
Cannot use Bytes type propperly

### ISSUE THREAD

**Issue by aristidesstaffieri** [stellar/soroban-tools#436]
### What version are you using?
0.6.0

### What did you do?
I have deployed an instance of the token interface on futurenet, and I'm trying to call initialize with [this fn sig](https://github.com/stellar/soroban-examples/blob/main/token/src/contract.rs#L14). The last two args are Bytes that I believe are supposed to be the hex string representation of the value. I am trying the following and am wondering why it wouldn't be able to parse the --symbol arg but had no problem with the --name arg, both are hex strings of the value that I want to pass for name and symbol.
```
$ soroban contract invoke \
    --id 6885ed6b904a6cead909c99cdc482ffa4a74c181803f678cacbfbe831e9efac6 \
    --secret-key S... \
    --rpc-url http://localhost:8000/soroban/rpc \
    --network-passphrase 'Test SDF Future Network ; October 2022' \
    --fn initialize \
    -- \
    --admin GCGORBD5DB4JDIKVIA536CJE3EWMWZ6KBUBWZWRQM7Y3NHFRCLOKYVAL \
    --decimal 7 \
    --name 467265696768746572205465737420546f6b656e \
    --symbol 465454
error: parsing argument symbol: invalid type: integer `465454`, expected string or map
```

### What did you expect to see?
A successful transaction for initializing this contract

### What did you see instead?
cli cannot parse the hex string provided for the bytes arg called symbol, it seems to be fine with the hex string for bytes for the arg called name.

**Comment by willemneal** [comment]
Hmm interesting.  Does it still happen if you change symbol to `symbol_b`?  We do have a test for bytes. But I'll add one with an arg named `symbol`.

**Comment by aristidesstaffieri** [comment]
> Hmm interesting. Does it still happen if you change symbol to `symbol_b`? We do have a test for bytes. But I'll add one with an arg named `symbol`.

Not sure I understand, do you mean change the arg name to `symbol_b`, just to clarify this is the symbol used to identify the token in this instance, not a Symbol type but a Bytes type.

**Comment by bjorn** [comment]
Just ran into this on RPCiege day 3, which required sending in a symbol of which the hash included a certain pattern. I was unable to pass any symbols on the CLI that consisted of just numbers, likely because they were being parsed as integers and then failed a type check later on when invoking the contract's function:
```
$ soroban contract invoke --id ... --source ... --rpc-url https://rpc-futurenet.fastcheapandoutofcontrol.com:443 --network-passphrase 'Test SDF Future Network ; October 2022' -- game_3 --symbol 123 --_nft_dest ...
error: parsing argument symbol: invalid type: integer `123`, expected string or map
```

**Comment by willemneal** [comment]
Did you try the symbol in quotes? "123"?

**Comment by bjorn** [comment]
> Did you try the symbol in quotes? "123"?

Yes, but that didn't make a difference because the shell would strip them. I had not tried the `'"123"'` variant though.
